verification/scripts/countlines.py

```diff
@@ -0,0 +1,58 @@
+import os
+from os import path
+import re
```

Comment:
fine by me as long as we add a comment saying what currently does not work(e.g., multi-line comments, hybrid lines) and how we count line (e.g., how .gobra files are handled, that we only count non empty, non commented out lines)